recipes/base_config.rb

```diff
@@ -16,7 +16,6 @@
 # limitations under the License.
 
 include_recipe "aws-parallelcluster::setup_envars"
-include_recipe 'aws-parallelcluster::base_install'
```

Comment:
I'm wondering if setup_envars recipe included in base_install is required in base_config too.

Reply:
As in you think we might be able to also remove line 18 above as well?